examples/js/controls/PointerLockControls.js

```diff
@@ -83,28 +116,33 @@ THREE.PointerLockControls = function ( camera, domElement ) {
 
 	this.getObject = function () {
 
-		return yawObject;
+		console.warn( 'THREE.PointerLockControls: getObject() has been deprecated.' );
+
+		return scope.object;
 
 	};
 
-	this.getDirection = function () {
+	this.getDirection = function ( v ) {
```

Comment:
I noticed that getDirection function is not used anywhere. Perhaps it is safe to remove?

Reply:
Other users may use it.